### ISSUE TITLE
NPCs spawn endlessly

### ISSUE THREAD

**Issue by simonelambiase** [DRE2N/DungeonsXL#830]
**Description**

The problem occurs with the use of signs to spawn mobs, in particular with a sign to spawn a certain NPC, when the instance is started errors in the console begin to arrive continuously and the NPC seems to continue to regenerate.

**Reproduce**

1. Create a new dungeon with the base signs
2. Create a sign with a npc and start the dungeon
3. The npc will start to spawn at infinite time

**Expected behavior**

The NPC should spawn only once instead of spawning multiple times

**Screenshots / GIFs / videos**

Log pastebin: 
https://pastebin.com/x8b5k9sh

Video link: 
https://youtu.be/hziaMJWx8yo

Screenshot: 
https://user-images.githubusercontent.com/9357251/85342936-fd683280-b4eb-11ea-8c00-6013ca4fceab.png

(If applicable, add screenshots to help explain your problem.)

**Relevant configuration files**

...

**Comment by Sataniel98** [comment]
Duplicate of #807
